internal/archive/archive.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/canonical/chisel/internal/control"
 	"github.com/canonical/chisel/internal/deb"
 	"github.com/canonical/chisel/internal/pgputil"
+	"github.com/canonical/chisel/internal/setup"
```

Comment:
That relationship doesn't seem right.  What is an "archive", what is a "setup", and how does it make sense for the former to know about the latter?  If we don't keep abstraction lines in mind, it all becomes an amorphous blob soon.

Reply:
If we want to make it unique, then at least invert the relationship because `archive` is more lower level and should be generic, `setup` is something unique for Chisel so that is why the constants should be defined here.

Reply:
Done in https://github.com/canonical/chisel/pull/167/commits/e8be8301bed4c96cb4e44b648790c7df091479e8.